check_dvcs.py

```diff
@@ -58,8 +57,11 @@ def delete_previous_comments(comments_urls: list[str]) -> None:
         raise CommandException('\n'.join(comments_that_failed_to_delete))
 
 
-def does_string_contain_jira(string_to_match: str) -> Optional[str]:
-    pr_title_re = re.compile(f"({_AAP_RE}|{_NO_JIRA_MARKER})")
+def does_string_contain_jira(string_to_match: str, allow_no_jira: bool) -> Optional[str]:
+    if allow_no_jira and "NO_JIRA" in string_to_match:
+        print(f"NO_JIRA found in {string_to_match}")
+        return "NO_JIRA"
+    pr_title_re = re.compile(f"({_AAP_RE})")
```

Comment:
Do you need the parens in here since we no longer do a `|`?

Reply:
Yeah we still need to capture since we're pulling out the first capture group below on line 72

Reply:
I think there is always a group 0 which is the entire string match isn't there?

Reply:
```
>>> import re
>>> pr_title_re = re.compile("(AAP-[0-9]+)")
>>> pr_title_re.search('foo bar AAP-1234 hey').groups()
('AAP-1234',)
>>> pr_title_re = re.compile("AAP-[0-9]+")
>>> pr_title_re.search('foo bar AAP-1234 hey').groups()
()
```